src/main/java/net/pms/network/Request.java

```diff
@@ -327,10 +329,16 @@ public void answer(
 							// XXX external file is null if the first subtitle track is embedded:
 							// http://www.ps3mediaserver.org/forum/viewtopic.php?f=3&t=15805&p=75534#p75534
 							if (sub.isExternal()) {
-								inputStream = new FileInputStream(sub.getExternalFile());
+								if (sub.getType() == SubtitleType.SUBRIP && mediaRenderer.isRemoveTagsFromSRTsubs()) { // remove tags from .srt subs when renderer doesn't support them
+									inputStream = new FileInputStream(SubtitleUtils.removeTagsFromSubs(sub.getExternalFile()));
+								} else {
+									inputStream = new FileInputStream(sub.getExternalFile());
+								}
+							} else {
+								LOGGER.trace("Not loading external subtitles because they are not external: " + sub);
 							}
-						} catch (NullPointerException npe) {
-							LOGGER.trace("Could not find external subtitles: " + sub);
+						} catch (IOException npe) {
```

Comment:
@valib Catching NPE's is an absolute no-no as I see it, it should be null checked instead. But it can happen here, as the comments above explains. PMS forums are down, so I can't check the link, but it's easy enough to accept that ```sub.getExternalFile()```. This was added in 27a277b4f300961e266eb3a30d664c2d5af1ecef.